codegen/src/main/twirl/templates/JavaServer/Handler.scala.txt

```diff
@@ -15,6 +15,7 @@ import akka.japi.Function;
 import akka.http.javadsl.model.*;
 import akka.actor.ActorSystem;
 import akka.stream.Materializer;
```

Comment:
Can we drop the `import akka.stream.Materializer`, then?

Reply:
In this case we still have an implementation of `partial` that allows passing in the `Materializer`.

I'm not sure we already want to go so far as to make it impossible to bring your own Materializer, just have it less in-your-face.

(i *think* fatal warnings would have made it obvious if we had unused imports even in generated code, but haven't double-checked ;) )